platform/core/common/styles/_basicElements.scss

```diff
@@ -19,56 +19,56 @@
   font-family: 'Roboto';
   font-style: normal;
   font-weight: 300;
-  src: local("Roboto Light"), local("Roboto-Light"), url("../../../../../roboto-fontface/fonts/roboto/Roboto-Light.ttf") format("truetype");
+  src: local("Roboto Light"), local("Roboto-Light"), url("../../../../../node_modules/roboto-fontface/fonts/roboto/Roboto-Light.woff2") format("woff2");
```

Comment:
really should parameterize the folder location to the roboto fonts to allow consumers of nifi-fds the ability to provide their folder in an override.

define it somewhere...
```
$fds-font-path:    "../../../../../node_modules/roboto-fontface/fonts/roboto" !default;
```

then use it here:

```
src: local("Roboto Light"), local("Roboto-Light"), url("#{fds-font-path}/Roboto-Light.woff2") format("woff2");
```

Reply:
I created this issue to address this problem: https://issues.apache.org/jira/browse/NIFI-6291